app/controllers/profiles_controller.rb

```diff
@@ -9,7 +9,15 @@ class ProfilesController < ApplicationController
 
   def show
     @user = User.find_by_slug!(params[:id])
```

Comment:
Would it work to stick a confirmed scope here so we don't need to manually check the user?

Reply:
That makes sense. I've made the change for this. Thanks.